libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -92,6 +94,7 @@ const AP_Param::GroupInfo AP_BLHeli::var_info[] = {
     // @Units: Hz
     // @Range: 0 500
     // @User: Standard
+    // @RebootRequired: True
```

Comment:
This can be changed dynamically if non-zero (which it is by default)

Reply:
ok, removed the reboot required, txs!